src/introduction/a-path-to-pos-zcash.md

```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
 
-We are refining the design of TFL with several design goals. As we refine the design and make various trade-offs, we may not be able to achieve all of these goals.
+### Considering Hybrid Multi-Step approach (vs Single-Step):
 
-- We want minimal-to-no disruption for existing wallet use cases and UX. For example, nothing should change for the user flows for storing or transferring funds, the format of addresses, etc.
-- We want a security analysis of the proposed protocol to be as simple as possible given existing security analyses of current Zcash.
-- We want to enable new use cases around PoS that allow mobile shielded wallet users to earn a return on delegated ZEC.
-- We want to enable trust-minimized bridges and other benefits by providing a protocol with [assured finality](../terminology.md#definition-assured-finality).
-- We want to improve the _modularity_ of the consensus protocol. That is, it should be possible to understand some consensus properties only given knowledge of a "component" of the protocol, and it should be possible to implement consensus rules in modular code components with clean interfaces.
+Note: TFL is one instance of a multi-step approach.
 
-We will be refining these goals and potentially adding more as we continue to develop this proposal.
+#### Pros
+- We can hopefully be less disruptive across the ecosystem so that there are fewer snags and disruptions with each step.
+- If there is a show-stopping flaw in any step, the fall-back possibility seems more plausible. For example, if there is a show-stopper when transitioning from PoW to [PoW+TFL](../terminology.md#definition-pow-tfl), falling back to pure PoW seems more feasible, since both protocols rely on mainnet PoW infrastructure, so those participants will be present in either case.
+- Retaining miners during a hybrid phase: while it is true that a hybrid protocol will lower miner revenue (since we aim to maintain the issuance schedule constraints), there is also more possibility and likelihood of keeping some of these users engaged. For example, they may begin participating in staking services (either as delegators or as infrastructure operators). If that is successful, then they’re also more likely to remain engaged in the subsequent transition to pure PoS.
+- This general approach was demonstrated successfully by Ethereum, which is the largest or second largest cryptocurrency network for several important metrics (e.g. market cap, fees paid, user and developer activity, …). So we know this can be done well without disruptions.
 
-With the motivation and goals for a first-step hybrid PoW/PoS protocol in mind, we introduce the specifics of TFL in the next subchapter.
+#### Cons
+- The intermediate hybrid step will be a more novel and less well understood protocol. (It will necessarily be fairly different from Ethereum’s Beacon chain era.)
```

Comment:
Why will it necessarily be fairly different from Ethereum's Beacon chain era? Despite the design work on Crosslink which is indeed different, I think we still haven't excluded the option to "just replicate as closely as possible what Ethereum did".

Reply:
There are many many details for Beacon Chain that would be Ethereum specific. Some user-facing examples:

- I don't know if the beacon chain provided finality!
- Users could deposit to the beacon chain for staking prior to the beacon chain launching (I believe).
- Users could stake and run beacon chain validators without any ability to withdraw to L1 for a long time (~18 months).
- Users could deposit to stake the Beacon chain by depositing into a specific contract with a well-known address, so the UX was specific to Ethereum's contract addresses.
- I don't know if non-beacon nodes had to refer to beacon blocks, as Crosslink does.
- The staking rates, slashing amounts, slashing triggers, etc… were all likely to be different from our design.
- I believe it may have activated based on amount of stake deposited, whereas we may want to just have a pre-defined activation height? (Open question.)

So basically I think there are many unknowns (for me) and I simply don't want to lead any readers into _expecting_ any feature/behavior parity with the Beacon Chain approach.

We can always explicitly decide we want to adopt similar design goals! I just don't want to pre-commit to parity.

(BTW, one aspect I really liked is the despits-before-activate because that serves as an unambiguous skin-in-the-game permissionless signal of confidence in the new protocol feature.)